cmd/serverless/main.go

```diff
@@ -201,8 +201,11 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		log.Error("No API key configured, exiting")
 	}
 	config.Datadog.SetConfigFile(datadogConfigPath)
+	// Load datadog.yaml file into the config, so that metricAgent can pick these configurations
+	if _, err := config.Load(); err == nil {
```

Comment:
```suggestion
	if _, err := config.Load(); err != nil {
```

I assume we only want to log the error if there's an error.

Reply:
Good catch